### ISSUE TITLE
Error 404 when the path in the base URL is not empty

### ISSUE THREAD

**Issue by m4r1vs** [mk-5/fjira#122]
**Desktop (please complete the following information):**
 - OS: Linux 6.11.6-arch1-1
 - Terminal Alacritty
 - Architecture: x86
 - Fjira version: From source (commit db02e39)

**Describe the bug**
When the API URL includes a path, after initializing `fjira`, I do not get a
list of projects but a `404` instead.

**To Reproduce**

Set config like this
```yaml
current: default
workspaces:
    default:
        jiraRestUrl: https://tgipm.informatik.uni-hamburg.de/jira/
        jiraToken: TOKEN
        iraUsername: USERNAME
        jiraTokenType: personal token
```

Run `fjira`. Notice the `/jira` in the path of `jiraRestUrl`. The error is caused by `https://tgipm.informatik.uni-hamburg.de/restApi` being called instead of `https://tgipm.informatik.uni-hamburg.de/jira/restApi`. 

**Expected behavior**
See project list.

**Actual Behavior**
Get Error 404

**Additional notes**
- [PR](https://github.com/mk-5/fjira/pull/120)

**Comment by mk-5** [comment]
@m4r1vs thanks for a report! I will check PR right away

**Comment by mk-5** [comment]
@m4r1vs I've merged it - but I want to add some unit test for that paths combine logic before I release it

**Comment by mk-5** [comment]
fixed in [1.4.3](https://github.com/mk-5/fjira/releases/tag/1.4.3)
